test/integration/specialist_document_test.rb

```diff
@@ -107,8 +107,8 @@ def test_meta(component)
   test "renders a nested contents list" do
     setup_and_visit_content_item('aaib-reports')
 
-    assert page.has_css?(".dash-list")
-    within ".dash-list" do
+    assert page.has_css?(".contents-list.contents-list-nested")
```

Comment:
Is this line necessary? Presumably test will fail anyway if not present, and this line isn't repeated at line 121 where another test does exactly the same thing. I guess the test might fail in a different way if the line is simply omitted, but if that's the case then it should also occur before line 121 as well.